src/raqm.c

```diff
@@ -243,6 +249,10 @@ raqm_create (void)
 
   rq->flags = RAQM_FLAG_NONE;
 
+  rq->line_width = 2147483647;
```

Comment:
@khaledhosny the maximum integer (so that the default width is infinity), I was intending but forgot to change that to something proper.